packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -120,6 +121,7 @@ const AccountSwitcher = ({
         closeAccountsDialog();
         if (account_loginid === loginid) return;
         await switchAccount(loginid);
+        RudderStack.setAccountType(loginid.substring(0, 2));
```

Comment:
```suggestion
        RudderStack.setAccountType(loginid.replace(/\d/g, ''));
```
Can we use regex to get the account type from loginid instead of fixed characters? This will pick up CR, MLT, etc..

Reply:
Actually, marketing team wants to get 2 first characters only, I believe it makes it easier for them to analyze the data this way.